server/src/app/api/schemas.py

```diff
@@ -23,15 +23,11 @@ class _Id(BaseModel):
 
 
 # Metrics
-class MetricIn(_CreatedAt, _Id):
+class MetricIn(_Id):
     cpu_temperature_C: float = Field(None, example=17.3)
     mem_available_GB: float = Field(None, example=1.23)
     cpu_usage_percent: float = Field(None, example=51.8)
 
 
 class MetricOut(MetricIn):
```

Comment:
As long as timestamp are handled on API, do we need a timstamp as an input ? (cf our previous discussion about it )

Reply:
No, I meant: you HAVE to add them so that the API sets it itself :sweat_smile: 

The other solution is that: whatever the database you will send this to, will need to add its timestamp. If that's OK with you, we can indeed merge.

Reply:
My bad, that is way I added _created_at in the first place ^^
 
I have fixed it, you can and approve and this PR will be ready to merge :)